jfr-daemon/src/main/java/com/newrelic/jfr/daemon/EnvironmentVars.java

```diff
@@ -24,6 +24,8 @@
   public static final String PROXY_PASSWORD = "PROXY_PASSWORD";
   public static final String PROXY_SCHEME = "PROXY_SCHEME";
   public static final String THREAD_NAME_PATTERN = "THREAD_NAME_PATTERN";
+  public static final String HARVEST_INTERVAL = "HARVEST_INTERVAL";
+  public static final String QUEUE_SIZE = "QUEUE_SIZE";
```

Comment:
Can you add these configuration options to the [table in the readme](https://github.com/newrelic/newrelic-jfr-core#configuration-for-standalone-usage)? For that matter, I doesn't look like the `THREAD_NAME_PATTERN` option ever got added as either. It would be great to add that as well.